### PR TITLE
[Docs] Add Chinese version of customize_models.md

### PR DESCRIPTION
**Pull request by Abyssaledge**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add Chinese version `customize_models.md`

## Modification
Add Chinese version `customize_models.md`

## BC-breaking (Optional)
Does the modification introduce changes that break the back compatibility of the downstream repos?
No

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zytx121** [comment]
Hi @Abyssaledge 
Please fix the lint first.
You can refer to https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md for proper use of pre-commit.

**Comment by Abyssaledge** [comment]
@zytx121 Thanks for reminding, I believe the linting problem is fixed now.

File: docs/zh_cn/tutorials/customize_models.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+我们大致将模型组件分为了 5 种类型。
 
-- backbone: usually an FCN network to extract feature maps, e.g., ResNet, Swin.
-- neck: the component between backbones and heads, e.g., FPN, ReFPN.
-- head: the component for specific tasks, e.g., bbox prediction.
-- roi extractor: the part for extracting RoI features from feature maps, e.g., RoI Align Rotated.
-- loss: the component in head for calculating losses, e.g., FocalLoss, GWDLoss, and KFIoULoss.
+- 主干网络 (Backbone): 通常是一个全卷积网络 (FCN)，用来提取特征图，比如残差网络 (ResNet)。也可以是基于视觉 Transformer 的网络，比如 Swin Transformer 等。
+- Neck: 主干网络和任务头 (Head) 之间的连接组件，比如 FPN, ReFPN。
+- 任头务 (Head): 用于某种具体任务（比如边界框预测）的组件。
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_models.md:7]: 任务头

File: docs/zh_cn/tutorials/customize_models.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+### 添加新的主干网络
 
-Here we show how to develop new components with an example of MobileNet.
+这里，我们以 MobileNet 为例来展示如何开发新组件。
 
-#### 1. Define a new backbone (e.g. MobileNet)
+#### 1. 定义一个新的主干网络（以 MobileNet 为例）
 
-Create a new file `mmrotate/models/backbones/mobilenet.py`.
+新建文件 `mmrotate/models/backbones/mobilenet.py`.
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_models.md:19]: 最后的标点可以用中文的句号

File: docs/zh_cn/tutorials/customize_models.md [hunk 3]
```diff
@@ -64,11 +64,11 @@ model = dict(
...
 ```
 
-### Add new necks
+### 添加新的 Neck
 
-#### 1. Define a neck (e.g. PAFPN)
+#### 1. 定义一个 Neck（以 PAFPN 为例）
 
-Create a new file `mmrotate/models/necks/pafpn.py`.
+新建文件： `mmrotate/models/necks/pafpn.py`.
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_models.md:71]: 这里的冒号可以省略，最后的句号也可以改为中文句号。

File: docs/zh_cn/tutorials/customize_models.md [hunk 4]
```diff
@@ -118,13 +118,12 @@ neck=dict(
...
-### Add new heads
+### 添加新的 Head
 
-Here we show how to develop a new head with the example of [Double Head R-CNN](https://arxiv.org/abs/1904.06493) as the following.
+这里，我们以 [Double Head R-CNN](https://arxiv.org/abs/1904.06493) 为例来展示如何添加一个新的 Head。
 
-First, add a new bbox head in `mmrotate/models/roi_heads/bbox_heads/double_bbox_head.py`.
-Double Head R-CNN implements a new bbox head for object detection.
-To implement a bbox head, basically we need to implement three functions of the new module as the following.
+首先，添加一个新的 bbox head 到 `mmrotate/models/roi_heads/bbox_heads/double_bbox_head.py`.
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_models.md:125]: 这里也可以使用中文句号。

File: docs/zh_cn/tutorials/customize_models.md [hunk 5]
```diff
@@ -244,24 +243,23 @@ class DoubleHeadRoIHead(StandardRoIHead):
...
-Alternatively, the users can add
+另外，用户也可以添加
 
 ```python
 custom_imports=dict(
     imports=['mmrotate.models.roi_heads.double_roi_head', 'mmrotate.models.bbox_heads.double_bbox_head'])
 ```
 
-to the config file and achieve the same goal.
+到配置文件中来实现同样的功能.
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_models.md:255]: 中文句号

**Comment by Abyssaledge** [comment]
@jbwang1997 感谢提醒，标点问题已经修改。

**Review by jbwang1997 — APPROVED** [review]
我这里没有什么问题了。

**Comment by jbwang1997** [comment]
Kindly ping @zytx121 to have a look.

**Review by yangxue0827 — APPROVED** [review]
LGTM

**Pull request merged**